backend/resources/auth.py

```diff
@@ -45,6 +45,10 @@ def post(self):
         username = result["username"]
         password = result["password"]
 
+        if not (username and password):
+            return ({"msg": "Username and password required"},)
```

Comment:
this need to be
```
return ({"msg": "Username and password required"}, HTTPStatus.BAD_REQUEST)
```
otherwise `HTTPStatus.BAD_REQUEST` will never be returned.

Reply:
Here you go: https://github.com/CodeForPoznan/codeforpoznan.pl_v3/pull/240